spec/system/planning_applications/sitemap_spec.rb

```diff
@@ -251,7 +251,7 @@
       end
 
       let(:new_geojson) do
-        {type: "FeatureCollection", features: [new_geojson_feature]}.to_json
+        {"type" => "FeatureCollection", "features" => [new_geojson_feature]}
```

Comment:
This is used on line 258 which will set a hash string - either call `to_json` here or on line 258